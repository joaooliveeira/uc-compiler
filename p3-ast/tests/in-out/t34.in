int N = (int) 100.0;
