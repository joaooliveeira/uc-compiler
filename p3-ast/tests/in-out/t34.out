Program:
    GlobalDecl:
        Decl: ID(name=N)
            VarDecl:
                Type: int @ 1:1
            Cast: @ 1:9
                Type: int @ 1:10
                Constant: float, 100.0 @ 1:15
